docs/medical-api/more-info/pagination.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Pagination"
+description: "Retrieving larger amounts of data."
+---
+
+Some of Metriport's endpoints are paginated: they return a limited amount of items each time they are requested.
```

Comment:
nit: prefer `-` over `:` in this sentence

Reply:
done